dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-dependent.ts

```diff
@@ -308,7 +308,18 @@ export function useDependent(model: { [field: string]: any }): IJsonItem[] {
                   processCode
                 )
                 item.depTaskCode = 0
-              }
+              },
+              renderOption: ({
+                node,
+                option
+              }: {
+                node: VNode
+                option: SelectOption
+              }) =>
+                h(NTooltip, null, {
+                  trigger: () => node,
+                  default: () => option.label
+                })
```

Comment:
Here the Ellipsis is better than Tooltip because there is no need to prompt when the name is not overflowing.